src/Compilers/CSharp/Portable/Symbols/FieldSymbol.cs

```diff
@@ -387,7 +387,18 @@ public virtual bool IsTupleField
         /// <summary>
         /// Returns True when field symbol is not mapped directly to a field in the underlying tuple struct.
         /// </summary>
-        internal virtual bool IsVirtualTupleField
+        public virtual bool IsVirtualTupleField
+        {
+            get
+            {
+                return false;
+            }
+        }
+
+        /// <summary>
+        /// Returns true if this is a field representing a Default element like Item1, Item2...
+        /// </summary>
+        public virtual bool IsDefaultTupleElement
```

Comment:
`IsVirtualTupleField` and `IsDefaultTupleElement`: do `Element` and `Field` mean the same here? If so, we should choose one. (My vote is for `Field`.)